homeassistant/components/ios/storage.py

```diff
@@ -25,28 +24,22 @@ def __init__(self, hass: HomeAssistant) -> None:
         self.data: dict[str, Any] = {}
         self.subscriptions: dict[str | None, list[Callable[[], None]]] = {}
         self._loaded = False
-        self._load_lock = asyncio.Lock()
 
     async def async_load(self) -> None:
         """Load the data from storage."""
         if self._loaded:
```

Comment:
I'd remove this check and attribute too. We should only load the store once during the Home Assistant application lifetime in the current design.